src/CommonITILObject.php

```diff
@@ -9170,11 +9172,21 @@ public static function getDataToDisplayOnKanban($ID, $criteria = [])
             }
         }
 
+        $category_names = [];
         foreach ($common_itil_iterator as $data) {
+            if (!isset($category_names[$data['itilcategories_id']])) {
```

Comment:
`array_key_exists` is more precise than `isset`, as `isset` would return false when key exists but value is null.
```suggestion
            if (!array_key_exists($data['itilcategories_id'], $category_names)) {
```